tuf/data/roles.go

```diff
@@ -214,6 +214,18 @@ type Role struct {
 	Paths []string `json:"paths,omitempty"`
 }
 
+func isValidRootRoleStructure(r RootRole, validKeys Keys) error {
```

Comment:
I know we've talked about it before, did we decide we should enforce that there must be enough keys to cover the Threshold?